docs/workforce-management-framework/topics/con-release-notes.adoc

```diff
@@ -0,0 +1,25 @@
+= {PROJECT_NAME} {WFM-RC-Version} Release Notes
+
+== Overview
+
+{PROJECT_NAME} allows you to move from paper based forms and processes into complete Workflow management mobile solution.
```

Comment:
complete Workflow management mobile . Workflow is internal. `Process` will be better as it's more obvious.